### DISCUSSION TITLE
The new version of youtubedownloader is not working

### DISCUSSION THREAD

**Discussion by supra2511** [Tyrrrz/YoutubeDownloader#520] [General]
Dear GitHub

I replaced the old version with the new version v1.13.2 and v1.13.3 however the both new version are not working
How to install and use the new version?

when I first click the application, the below message box appeared
![Screenshot 2024-09-28 205852](https://github.com/user-attachments/assets/1c62fdb1-dad2-472b-9f36-9ab7ccc5829b)

the list box of download items appeared then disappeared

let me know how to solve the problem

thank so much that I have used the application for years with no problem
however today, I got a problem

Best regards

**Comment by Tyrrrz** [comment]
That just means you try run the app in a location where it cannot save the settings. Either move the app to a different directory or run it as administrator.

**Reply by supra2511** [reply]
thanks Tyrrrz
As you advised I moved the app folder from Program Files folder to other folder
It works now

**Comment by Reynolds556** [comment]
Maybe you can try [Keepvid downloader](https://keepvid.ch/).

I've downloaded several YouTube videos with this tool. It's easy to use, only copy the video URL and then convert it to MP4 file.

**Reply by verdure1213** [reply]
why my mp4 has no sounds
